No such property: application for class: grails.buildtestdata.BuildTestDataGrailsPlugin

Using Grails `3.2.0.RC1` and `3.0.0` version of the plugin I get the following error during integration tests:

```groovy
Caused by: groovy.lang.MissingPropertyException: No such property: application for class: grails.buildtestdata.BuildTestDataGrailsPlugin
    at grails.buildtestdata.BuildTestDataGrailsPlugin$_closure1.doCall(BuildTestDataGrailsPlugin.groovy:32)
```

Full Stacktrace:

https://gist.github.com/rpalcolea/cf7be13d2cbbd803c05c4976540441f4

--- a/build-test-data/build.gradle
+++ b/build-test-data/build.gradle
@@ -16,7 +16,7 @@ plugins {
     id "com.jfrog.bintray" version "1.2"
 }
 
-version "3.0.0"
+version "3.0.1"
 group "org.grails.plugins"
 
 apply plugin: 'maven-publish'

--- a/build-test-data/src/main/groovy/grails/buildtestdata/BuildTestDataGrailsPlugin.groovy
+++ b/build-test-data/src/main/groovy/grails/buildtestdata/BuildTestDataGrailsPlugin.groovy
@@ -29,11 +29,11 @@ class BuildTestDataGrailsPlugin {
 
     def doWithApplicationContext = { appCtx ->
         DomainUtil.setApplication(appCtx.grailsApplication)
-        decorateDomainClasses(application.domainClasses)
+        decorateDomainClasses(appCtx.grailsApplication.domainClasses)
     }
 
     def onChange = { event ->
-        decorateDomainClasses(application.domainClasses)
+        decorateDomainClasses(grailsApplication.domainClasses)
     }
 
     private decorateDomainClasses(domainClasses) {

